### PR TITLE
Remove bv divide by zero option

### PR DESCRIPTION
**Pull request by ajreynol**
This is required to avoid failures in the planned refactoring of `check-models`.

This removes the SMT-LIB 2.5 semantics option for bvdiv/bvrem.

It still remains to merge the BITVECTOR_UDIV / BITVECTOR_UDIV_TOTAL kinds, calling the total version "BITVECTOR_UDIV", and analogously for UREM.

FYI @barrettcw

### CODE REVIEW

**Review by yoni206 — COMMENTED** [review]
Great that you are doing this :) Simplifies these parts of the code. I have two questions inline.

File: src/theory/bv/theory_bv_rewriter.cpp [hunk 1]
```diff
@@ -440,21 +440,13 @@ RewriteResponse TheoryBVRewriter::RewriteNeg(TNode node, bool prerewrite) {
 RewriteResponse TheoryBVRewriter::RewriteUdiv(TNode node, bool prerewrite){
   Node resultNode = node;
 
-  if(node[1].isConst() && node[1].getConst<BitVector>().getValue() != 0) {
-    return RewriteUdivTotal(node, prerewrite);
-  }
-
-  return RewriteResponse(REWRITE_DONE, resultNode); 
+  return RewriteUdivTotal(node, prerewrite);
```
[the remark this answers is not in the document]
Comment (ajreynol) [reply src/theory/bv/theory_bv_rewriter.cpp:443]: BITVECTOR_UDIV and BITVECTOR_UDIV_TOTAL have the exact same semantics now always (which is the previous semantics of BITVECTOR_UDIV_TOTAL), thus we should always call the rewriter for BITVECTOR_UDIV_TOTAL for both now.

[the remark this answers is not in the document]
Comment (ajreynol) [reply src/theory/bv/theory_bv_rewriter.cpp:443]: Also note that this code will be deleted on the PR that merges the two operators.

**Review by yoni206 — APPROVED** [review]
Got it, thanks.

**Pull request merged**